OracleDatabase/SingleInstance/tests/runContainerTests193.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Should be `Copyright (c) 2020 Oracle and/or its affiliates.`

Reply:
Fixed